lib/src/controls/surfaces/flyout/flyout.dart

```diff
@@ -122,3 +122,106 @@ class FlyoutContent extends StatelessWidget {
     );
   }
 }
+
+/// A tile that is used inside of [FlyoutContent]
+class FlyoutListTile extends StatelessWidget {
+  /// Creates a flyout list tile.
+  const FlyoutListTile({
+    Key? key,
+    this.onPressed,
+    this.tooltip,
+    this.icon,
+    required this.text,
+    this.trailing,
+    this.focusNode,
+    this.autofocus = false,
+  }) : super(key: key);
+
+  final VoidCallback? onPressed;
+
+  final String? tooltip;
+  final Widget? icon;
+  final Widget text;
+  final Widget? trailing;
+
+  final FocusNode? focusNode;
+  final bool autofocus;
+
+  @override
+  Widget build(BuildContext context) {
+    return HoverButton(
+      key: key,
+      onPressed: onPressed,
+      focusNode: focusNode,
+      autofocus: autofocus,
+      builder: (context, states) {
+        final theme = FluentTheme.of(context);
+        final radius = BorderRadius.circular(4.0);
+
+        Widget content = Container(
+          decoration: BoxDecoration(
+            color: ButtonThemeData.uncheckedInputColor(theme, states),
+            borderRadius: radius,
+          ),
+          padding: const EdgeInsets.only(
```

Comment:
Should this potentially be `EdgeInsetsDirectional`?

Reply:
Nice catch!